edx_exams/apps/api/v1/tests/test_views.py

```diff
@@ -777,3 +782,163 @@ def test_access_granted_past_due_exam_no_attempt(self):
         response = self.get_exam_access(self.user, self.past_due_url)
         self.assertEqual(200, response.status_code)
         self.assert_valid_exam_access_ticket(response, self.user, self.past_due_exam)
+
+
+@ddt.ddt
+class ExamAttemptViewTest(ExamsAPITestCase):
```

Comment:
nit: since we're already testing the api functions separately this could get pared down a lot with some mocks if you wanted.

Reply:
ohhh interesting 🤔 I'll see what I can do